.github/workflows/wheels.yml

```diff
@@ -24,10 +24,13 @@ jobs:
         env:
           PYTHON: "py -${{ matrix.python }}-${{ matrix.wordsize }}"
         shell: bash
+        # Instead of letting setup.py install a newer numpy we install it here
+        # using the oldest supported version for ABI compatibility
         run: |
           set -ex
           ${PYTHON} -m pip install --upgrade pip
           ${PYTHON} -m pip install setuptools wheel cython
```

Comment:
Does cython pull in numpy? If so, maybe better to install oldest-supported-numpy first?

Reply:
Removed cython